bin/ch-test

```diff
@@ -102,12 +102,16 @@ builder_set () {
     fi
 }
 
-# Create CH_TEST_TARDIR, avoiding #347.
+# Create CH_TEST_IMGDIR, avoiding #347.
 dir_img_mk () {
-    if [[ ! -d $CH_TEST_IMGDIR ]]; then
-        printf "creating %s\n\n" "$CH_TEST_IMGDIR"
-        mkdir "$CH_TEST_IMGDIR"
-    fi
+    dir_img_rm
+    printf "creating %s\n\n" "$CH_TEST_IMGDIR"
+    mkdir "$CH_TEST_IMGDIR"
```

Comment:
Doesn't this mean the manual `run` phase following a manual `build` will always fail?